src/util/ui_message.cpp

```diff
@@ -47,13 +47,19 @@ ui_message_handlert::ui_message_handlert(
     break;
     
   case JSON_UI:
-    std::cout << "[\n";
+    {
+      std::cout << "[\n";
+      json_objectt json_program;
+      json_program["Program"] = json_stringt(program);
+      std::cout << json_program;
+    }
     break;
     
   case PLAIN:
     break;
     
-  default:;
+  default:
+    assert(false);
```

Comment:
My suggestion is to remove default: statements wherever possible as the compiler will warn you about missing cases.